tests/sparsezoo/analysis/helpers.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Helper fixtures and functions for testing sparsezoo.analysis
+"""
+
+import onnx
+import pytest
+
+from sparsezoo import Zoo
+from sparsezoo.analysis import ModelAnalysis
+
+
+__all__ = [
+    "model_paths",
+    "get_test_model_names",
+    "get_expected_analysis",
+    "get_generated_analysis",
+    "get_model_onnx",
+    "get_model_and_node",
+]
+
+_MODEL_PATHS = {
+    "yolact_none": {
+        "stub": "zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none",
+        "truth": "tests/sparsezoo/analysis/yolact_none.yaml",
```

Comment:
should use relative path from this file (ie `__file__`) instead of assuming we are in the repo root